### PR TITLE
Start up update

### PR DESCRIPTION
**Pull request by klemie**
Review by doing the flowing work flow on your computer.

First connect to my database:
change your .env file to 
```.env
MONGO_USERNAME = SupportAdmin
MONGO_PASSWORD = UVR2015
MONGO_DB_STRING = aiw5o28

SERVER_PORT = 9090
```
then do the same as the video

https://github.com/UVicRocketry/Ground-Support/assets/45109638/bfc6a582-0a34-4a09-b4a7-4c3169e80840

### CODE REVIEW

File: client/src/views/active/flight-view.tsx [hunk 1]
```diff
@@ -1,38 +1,35 @@
...
+import { dataConfigParser } from '../../utils/data-parser';
 import _ from 'lodash';
 
 const ROUND_TO = 3;
 const FLOATING = true;
 
-export default function ModulesView() {
-	const [ dataConfig, setDataConfig ] = useState(dataConfigParser(require('../utils/sample-data.json')));
+export default function FlightView() {
+	// const [ dataConfig, setDataConfig ] = useState(dataConfigParser(require('../utils/sample-data.json')));
```
Comment (j7uu) [inline client/src/views/active/flight-view.tsx:12]: 🤔

File: client/src/views/active/flight-view.tsx [hunk 2]
```diff
@@ -1,38 +1,35 @@
...
+	// 		const field: Field = {
+	// 			module: module.module,
+	// 			fieldName: module.fields[index],
+	// 			fieldRange: module.fieldRanges[index],
+	// 			fieldValue: _.round(_.random(min, max, FLOATING), ROUND_TO)
+	// 		};
+	// 		fields.push(field);
+	// 	}
+	// 	return fields;
+	// });
```
Comment (j7uu) [inline client/src/views/active/flight-view.tsx:32]: 🤔🤔

File: client/src/views/active/flight-view.tsx [hunk 3]
```diff
@@ -56,7 +53,7 @@ export default function ModulesView() {
 				justifyContent="space-between"
 				alignItems="stretch"
 			>
-				{formattedModules.map((module) => {
+				{/* {formattedModules.map((module) => {
```
Comment (j7uu) [inline client/src/views/active/flight-view.tsx:56]: commented out??

File: client/src/views/active/startup-view.tsx [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+								</FormControl>
+							</Stack>
+                        </Stack>
+                    </Paper>
+				</Grid>
+				<Grid container direction={'row'} gap={3}>
+					<Grid item width={component?.Name === 'Big Red Bee' ? '100%' : '50%'}>
+						<TelemetryLog 
+							value="
+								salads
```
Comment (j7uu) [inline client/src/views/active/startup-view.tsx:144]: salad!! 🥗

File: client/src/views/rocket-details-view.tsx [hunk 5]
```diff
@@ -95,17 +97,19 @@ export default function RocketDetailsView(props: RocketDetailsProps) {
...
     const refresh = () => {
         _.delay(getRocket, 500);
-    }
-
+    };
     const getRocket = useCallback(async () => {
-        console.log(rocketId);
         const response = await api.getRocket(rocketId);
         const rocket = response.data as IRocketPopulated;
         console.log(response);
```
Comment (j7uu) [inline client/src/views/rocket-details-view.tsx:115]: console.log

**Review by j7uu — APPROVED** [review]

File: client/src/views/active/startup-view.tsx (hunk 4 above)
Comment (klemie) [reply client/src/views/active/startup-view.tsx:144]: yes ma'am

**Pull request merged**